README.md

```diff
@@ -1,18 +1,18 @@
-# TU Delft CS4110 Software Testing and Reverse Engineering
+# TU Delft CS4110: Software Testing and Reverse Engineering
 
-2015-2016 Q3 -- [TU Delft Study Guide](http://studiegids.tudelft.nl/a101_displayCourse.do?course_id=37845)
+2015-2016 Q3 — [TU Delft Study Guide](http://studiegids.tudelft.nl/a101_displayCourse.do?course_id=37845)
 
 ## Collaboration
 
 ### Communication
 
-We are using Slack! Use [cs4110.slack.com](https://cs4110.slack.com/) and sign-up with your tudelft.nl or utwente.nl email address.
+We are using Slack! Use [cs4110.slack.com](https://cs4110.slack.com/) and sign-up with your TU Delft or UTwente email address.
```

Comment:
Well, not only is @tudelft.nl allowed, as is @student.tudelft.nl, so the perfectly clear you'd at least want to fix that.